xdsl/dialects/csl/csl_stencil.py

```diff
@@ -74,7 +74,7 @@ class ExchangeDeclarationAttr(ParametrizedAttribute):
 
     name = "csl_stencil.exchange"
 
-    neighbor_param: ParameterDef[builtin.DenseArrayBase]
+    neighbor_param: ParameterDef[builtin.DenseArrayBase[builtin.IntegerType]]
```

Comment:
@n-io Would it be possible to check that the csl typing is correct, it all seemed reasonable to me from context but didn't have docs to compare against